src/lightning-mocks/input/input.js

```diff
@@ -49,9 +49,13 @@ export default class Input extends LightningElement {
 	@api timeAriaLabelledBy
 	@api timezone
 	@api type
-	@api validity
+    @api validity = {}
```

Comment:
I'd prefer this property to remain as-is without the instantiation to an empty object.

The main reason being it'd be quite a pain to maintain the property types and default values across all the default mocks. I can see these default values getting out of date without a reliable way to validate them. I'd prefer to just maintain the correct component shape and if you need a specific default you can set it in the test or override the mock.

Reply:
Got it - I was in 2 minds, but figured I'd set so you can let me know reason for why not :)